components/hubspot/hubspot.app.mjs

```diff
@@ -570,33 +549,36 @@ export default {
       }));
     },
     async searchFiles(params, $) {
-      return this.makeGetRequest("/files/v3/files/search", params, $);
+      return this.makeGetRequest(API_PATH.FILES, "/files/search", params, $);
     },
     async getSignedUrl(fileId, params, $) {
-      return this.makeGetRequest(`/files/v3/files/${fileId}/signed-url`, params, $);
+      return this.makeGetRequest(API_PATH.FILES, `/files/${fileId}/signed-url`, params, $);
     },
     async addContactsToList(listId, emails, $) {
       return this.makePostRequest({
-        endpoint: `/contacts/v1/lists/${listId}/add`,
+        api: API_PATH.CONTACTS,
+        endpoint: `/lists/${listId}/add`,
         data: {
           emails,
         },
       }, $);
     },
     async getAssociationTypes(fromObjectType, toObjectType, $) {
-      return this.makeGetRequest(`/crm/v4/associations/${fromObjectType}/${toObjectType}/labels`, null, $);
+      return this.makeGetRequest(API_PATH.CRMV4`/associations/${fromObjectType}/${toObjectType}/labels`, null, $);
```

Comment:
```suggestion
      return this.makeGetRequest(API_PATH.CRMV4, `/associations/${fromObjectType}/${toObjectType}/labels`, null, $);
```
Optional:

I would recommend just doing a minor refactor here where we can use only `makeRequest` instead of having specialized functions such as `getRequest` and `postRequest` by just passing `method` argument using `opts` and defaulting with `method = "get"` so `method` can be omited.